r2dbc-spi/src/main/java/io/r2dbc/spi/UserPasswordCredential.java

```diff
@@ -0,0 +1,9 @@
+package io.r2dbc.spi;
+
+/**
+ * UserPasswordCredentials used to authenticate with a database.
+ */
+public interface UserPasswordCredential extends Credential {
+    String user();
```

Comment:
Even though it's kind of obvious what the method does, every public API should have a JavaDoc IMHO. The JavaDoc offers a concrete specification of the method's behavior, and specifications are super important for SPIs.

Reply:
Agreed, resolved with commit 2b09f64